Stack/Opc.Ua.Core/Stack/Https/HttpsListener.cs

```diff
@@ -153,17 +159,35 @@ public void Close()
         {
             Stop();
         }
+
+        /// <summary>
+        /// Raised when a new connection is waiting for a client.
+        /// </summary>
+        public event EventHandler<ConnectionWaitingEventArgs> ConnectionWaiting;
+
+        /// <summary>
+        /// Raised when a monitored connection's status changed.
+        /// </summary>
+        public event EventHandler<ConnectionStatusEventArgs> ConnectionStatusChanged;
+
+        /// <remarks/>
+        public void CreateReverseConnection(Uri url, int timeout)
+        {
+            if (ConnectionStatusChanged == null || ConnectionWaiting == null)
+            {
+                throw new NotImplementedException();
```

Comment:
>throw new NotImplementedException(); [](start = 16, length = 36)

[Bug - High] Method needs to be implemented or can be removed

Reply:
reverse connect for https is not implemented... but the interface needs it

---
In reply to: [432394524](https://github.com/OPCFoundation/UA-.NETStandard/pull/998#discussion_r432394524) [](ancestors = 432394524)